README.md

```diff
@@ -138,30 +146,6 @@ There should be exactly one blank line between methods to aid in visual clarity
 
 If required, `@synthesize` and `@dynamic` should each be declared on new lines in the implementation.
 
-Colon-aligning method invocation should often be avoided.  There are cases where a method signature may have >= 3 colons and colon-aligning makes the code more readable. Please do **NOT** however colon align methods containing blocks because Xcode's indenting makes it illegible.
```

Comment:
I'm removing this because it breaks Xcode's auto-indentation and I'm not certain how to implement it in clang-format (which also always lines up colons). I also disagree that this makes things hard to read. For small blocks, it's fine. For large blocks, the block should likely be broken out into a method or function anyway.